app/src/main/java/chat/rocket/android/chatroom/adapter/ActionsAttachmentViewHolder.kt

```diff
@@ -0,0 +1,105 @@
+package chat.rocket.android.chatroom.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import chat.rocket.android.chatroom.uimodel.ActionsAttachmentUiModel
+import chat.rocket.android.emoji.EmojiReactionListener
+import chat.rocket.core.model.attachment.actions.Action
+import chat.rocket.core.model.attachment.actions.ButtonAction
+import kotlinx.android.synthetic.main.item_actions_attachment.view.*
+import androidx.recyclerview.widget.LinearLayoutManager
+import chat.rocket.android.R
+import chat.rocket.android.util.TimberLogger
+import com.facebook.drawee.backends.pipeline.Fresco
+import com.facebook.drawee.view.SimpleDraweeView
+import com.google.android.material.button.MaterialButton
+
+class ActionsAttachmentViewHolder(
+        itemView: View,
+        listener: ActionsListener,
+        reactionListener: EmojiReactionListener? = null,
+        var actionAttachmentOnClickListener: ActionAttachmentOnClickListener
+) : BaseViewHolder<ActionsAttachmentUiModel>(itemView, listener, reactionListener) {
+
+    init {
+        with(itemView) {
+            setupActionMenu(actions_attachment_container)
```

Comment:
`with` is useless here since there is only one block being used. Could be `itemView.setupActionMenu(actions_attachment_container)` instead.

Reply:
@filipedelimabrito Unable to access setupActionMenu method without `with(itemView)`

Reply:
`itemView.setupActionMenu(actions_attachment_container)` works.